worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +73,52 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	int turnsLeft = 4;
+	string wordGuessed;
+
+	cout << endl;
+	cout << "Please guess the password" << endl;
+
+	while (turnsLeft > 0)
+		{
+			if (turnsLeft > 0)
```

Comment:
Is this needed? If `turnsLeft > 0` is not true then you won't be in the `while` loop